File: seed/java-sdk/server-sent-events-openapi/with-wire-tests/src/main/java/com/snippets/Example29.java
package com.snippets;

import com.seed.api.SeedApiClient;
import com.seed.api.types.UnionStreamRequestBase;

public class Example29 {
    public static void main(String[] args) {
        SeedApiClient client =
                SeedApiClient.builder().url("https://api.fern.com").build();

        client.validateUnionRequest(UnionStreamRequestBase.builder()
                .prompt("prompt")
                .streamResponse(true)
                .build());
    }
}

File: seed/swift-sdk/server-sent-events-openapi/Tests/Snippets/Example29.swift
import Foundation
import Api

enum Example29 {
    static func snippet() async throws {
        let client = ApiClient(baseURL: "https://api.fern.com")

        _ = try await client.validateUnionRequest(request: UnionStreamRequestBase(
            streamResponse: true,
            prompt: "prompt"
        ))
    }
}
